app/controllers/workflow_credential_controller.rb

```diff
@@ -0,0 +1,37 @@
+class WorkflowCredentialController < ApplicationController
+  before_action :check_privileges
+  before_action :get_session_data
+
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  include Mixins::GenericListMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::GenericShowMixin
+  include Mixins::BreadcrumbsMixin
+
+  def self.model
+    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook
```

Comment:
We don't have a `Credential` subclass yet, depends on https://github.com/ManageIQ/manageiq-providers-workflows/pull/7

Reply:
I think we should not introduce a controller without the backing model, recommend either removing this controller or waiting for that PR to be merged

Reply:
Either way this `model` is wrong :wink: